File: src/main/kotlin/net/lyragames/practice/events/EventPlayer.java
package net.lyragames.practice.events;

import org.bukkit.entity.Player;

import java.util.UUID;

public class EventPlayer {

    private UUID uuid;
    private Player player;
    private PracticeEvent event;


    public UUID getUuid() {
        return this.uuid;
    }

    public Player getPlayer() {return this.player;}

    public PracticeEvent getEvent() {
        return this.event;
    }

    public EventPlayer(UUID uuid, PracticeEvent event) {
        this.uuid = uuid;
        this.player = player;
        this.event = event;
    }


}

File: src/main/kotlin/net/lyragames/practice/events/EventPlayer.kt
package net.lyragames.practice.events

import org.bukkit.Bukkit
import org.bukkit.entity.Player
import java.util.*


/**
 * This Project is property of Zowpy © 2022
 * Redistribution of this Project is not allowed
 *
 * @author Zowpy
 * Created: 3/8/2022
 * Project: lPractice
 */

class EventPlayer(val uuid: UUID) {

    val player: Player
      get() = Bukkit.getPlayer(uuid)

    var dead = false
    var offline = false

    var roundsPlayed = 0
}